README.md

```diff
@@ -410,10 +413,9 @@ Output:
 ```
 Allocation updated with txId : fb84185dae620bbba8386286726f1efcd20d2516bcf1a448215434d87be3b30d
 ```
-
 ##### Forbid Allocation
 
-There are various operations which you can forbid on an allocation. Forbid flag works with [update allocation](#update-allocation) command. Check its working first.
+There are various operations which you can forbid on an allocation. Forbid flag works with [update allocation](#update-allocation) command. Check its working before proceeding.
```

Comment:
Please remove `Check its working before proceeding`. In general, everything in the docs is expected to be checked by the doc team and should be left to readers to check.